tests/test_database_url.py

```diff
@@ -29,6 +29,20 @@ def test_database_url_properties():
     assert u.port == 123
     assert u.database == "mydatabase"
 
+    u = DatabaseURL(
+        "postgresql://username:password@/mydatabase?host=/var/run/postgresql/.s.PGSQL.5432"
+    )
+    assert u.dialect == "postgresql"
+    assert u.username == "username"
+    assert u.password == "password"
+    assert u.hostname == "/var/run/postgresql/.s.PGSQL.5432"
+    assert u.database == "mydatabase"
+
+    u = DatabaseURL(
```

Comment:
Is this documented anywhere? I couldn't find it.

Reply:
the `.s.PGSQL.5432` bit?

Reply:
Sorry picked the wrong line. I mean this  `unix_sock=/var/lib/postgresql`

Reply:
The `host=...` is obvious. But I couldn't find docs for `unix_sock`

Reply:
ah yep, that came from the google cloud sql docs: https://cloud.google.com/sql/docs/postgres/connect-run#python you can see it in the example snippet

Reply:
i was able to connect using this format of string:
```python
"postgresql+pg8000://user:pass@/db_name?unix_sock=/cloudsql/PROJECT_ID:REGION:INSTANCE_NAME/.s.PGSQL.5432"
```

Reply:
Great.